redata/statistics/tests.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+from scipy.stats import chi2
+
```

Comment:
I would change tests.py file name, we can most likely call it what it is mahalonobis_distance etc.